typescript/infra/config/environments/testnet3/validators.ts

```diff
@@ -1,23 +1,21 @@
-import { ChainName, chainMetadata } from '@hyperlane-xyz/sdk';
+import { CoreChainName, chainMetadata } from '@hyperlane-xyz/sdk';
 
 import {
   ChainValidatorConfigs,
   CheckpointSyncerType,
 } from '../../../src/config/agent';
 
-import { TestnetChains, environment } from './chains';
+import { environment } from './chains';
 
 const s3BucketRegion = 'us-east-1';
 
-const s3BucketName = <Chain extends ChainName>(
-  chainName: Chain,
-  index: number,
-) => `hyperlane-${environment}-${chainName}-validator-${index}`;
+const s3BucketName = (chainName: CoreChainName, index: number) =>
+  `hyperlane-${environment}-${chainName}-validator-${index}`;
 
-export const validators: ChainValidatorConfigs<TestnetChains> = {
+export const validators: ChainValidatorConfigs = {
   alfajores: {
     interval: 5,
-    reorgPeriod: chainMetadata.alfajores.blocks.reorgPeriod,
+    reorgPeriod: chainMetadata.alfajores.blocks!.reorgPeriod!,
```

Comment:
This is because these fields are now optional  on ChainMetadata but we'll always populate them for our core chains